artscraper/__init__.py

```diff
@@ -3,5 +3,7 @@
 from artscraper.googleart import GoogleArtScraper, random_wait_time
 from artscraper.wikiart import WikiArtScraper
 from artscraper.artist_links import get_artist_links
+from artscraper.artist_works_metadata import get_artist_works
 
-__all__ = ["GoogleArtScraper", "WikiArtScraper", "get_artist_links", "random_wait_time"]
+__all__ = ["GoogleArtScraper", "WikiArtScraper", \
```

Comment:
You should not need the "\" there.

```suggestion
__all__ = ["GoogleArtScraper", "WikiArtScraper",
```